jvm/src/test/kotlin/app/cash/trifle/TrifleTest.kt

```diff
@@ -85,6 +85,20 @@ internal class TrifleTest {
             .minusMillis(certHolder.notBefore.toInstant().toEpochMilli())
             .toEpochMilli()
         )
+      assertEquals(certTTL.toLong(), duration.toDays())
+    }
+
+    @Test
+    fun `test default validity period`() {
+      val endEntityDefault = certificateAuthority.createTestEndEntity("entity")
```

Comment:
call TestCertificateAuthority("issuingEntity", <some number here>) instead

Reply:
actually  i wanted to test the default call here and not including the validity was intentional.